src/openrct2/paint/Supports.cpp

```diff
@@ -595,8 +595,8 @@ bool wooden_a_supports_paint_setup(
             {
                 hasSupports = true;
                 paint_struct* ps = PaintAddImageAsOrphan(
-                    session, imageId, 0, 0, bBox.length.x, bBox.length.y, bBox.length.z, z, bBox.offset.x, bBox.offset.y,
-                    bBox.offset.z + z);
+                    session, ImageId::FromUInt32(imageId), { 0, 0, z }, { bBox.length.x, bBox.length.y, bBox.length.z },
```

Comment:
bBox.length is already a CoordsXYZ(D), right? So we could do this:

```suggestion
                    session, ImageId::FromUInt32(imageId), { 0, 0, z }, bBox.length,
```